actor/v7action/service_instance_details_test.go

```diff
@@ -151,7 +152,11 @@ var _ = Describe("Service Instance Details Action", func() {
 				Expect(fakeCloudControllerClient.GetServiceInstanceParametersArgsForCall(0)).To(Equal(serviceInstanceGUID))
 			})
 
-			When("the service instance is managed", func() {
+			It("does not get the service plan", func() {
+				Expect(fakeCloudControllerClient.GetServicePlanByGUIDCallCount()).To(Equal(0))
+			})
```

Comment:
I know this is to guard to make sure we only call the service plan endpoint in situations where upgrade info will be available, but does it make sense to move this closer to perhaps: `"upgrade is not available and the service does not have maintenance info"`?  I'm just weary that we might refactor how / what method gets called on the service, and this sort of floating `It` will remain technically valid, but others might not know why it's there or that it relates to upgrades.

Reply:
That makes sense. I'll update that in another PR.